MQuince/MQuince.Services.Tests/AppointmentTests.cs

```diff
@@ -0,0 +1,303 @@
+﻿using MQuince.Entities.Appointment;
+using MQuince.Entities.Users;
+using MQuince.Enums;
+using MQuince.Repository.Contracts;
+using MQuince.Services.Contracts.DTO.Appointment;
+using MQuince.Services.Contracts.Exceptions;
+using MQuince.Services.Contracts.IdentifiableDTO;
+using MQuince.Services.Contracts.Interfaces;
+using MQuince.Services.Implementation;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+
+namespace MQuince.Services.Tests.UnitTests
+{
+    public class AppointmentTests
+    {
+
+        IAppointmentService appointmentService;
+        IAppointmentRepository appointmentRepository = Substitute.For<IAppointmentRepository>();
+
+        public AppointmentTests()
+        {
+            appointmentService = new AppointmentService(appointmentRepository, null, null);
+        }
+
+        [Fact]
+        public void Constructor_when_give_repository_as_null()
```

Comment:
Implement a method for every argument, Constructor when given [argument name] null, and check if constructor throws ArgumentNullException if tested argument is null.